### PR TITLE
Integrate k2-asr

### PR DESCRIPTION
**Pull request by Triplecq**

### CODE REVIEW

File: pkg/k2-asr/src/transcribe.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        num_threads=1,
+        sample_rate=16000,
+        feature_dim=80,
+        decoding_method="greedy_search",
+        provider="cpu",
+    )
+    
+
+def transcribe(model, audio, config=None):
+    """Inference audio data using NeMo model
```
Comment (gunyarakun) [inline pkg/k2-asr/src/transcribe.py:32]: Not NeMo

File: pkg/k2-asr/src/transcribe.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    Args:
+        model (nemo.collections.asr.models.EncDecRNNTBPEModel): ReazonSpeech model
```
Comment (gunyarakun) [inline pkg/k2-asr/src/transcribe.py:35]: model must be a k2 model.

**Review by gunyarakun — COMMENTED** [review]
Find some NeMo

**Comment by fujimotos** [comment]
Merged via 1e1eb7852d04d554ac00713ddd85cb60517c9e97

**Pull request closed without merging**